apps/angularlogos/src/app/shared/logo-metadata.ts

```diff
@@ -8,10 +7,10 @@ export interface LogoMetadata {
   description?: string;
 }
 
-export interface LogoMetadataAll {
-  [filename: string]: LogoMetadata;
+export interface LogoMetadataEntity {
```

Comment:
The suffix `Entity` suggests that this is one entity and not the compound of entities. When reading `LogoMetadataEntity` I think of what we call `LogoMetadata` at the moment